hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -117,64 +117,66 @@ def create_phase_from_complex(incpx, outfloat, width):
     execute(f"cpx_to_real {incpx} {outfloat} {width} 4", uselogging=True)
 
 
-def get_water_mask(cc_mask_file, mwidth, lt, demw, demn, dempar):
+def get_water_mask(cc_file, mwidth, lt, demw, demn, dempar):
     """
-    create water_mask based on the cc_mask_file
+    create water_mask geotiff file based on the cc_file (float binary file)
     """
     with TemporaryDirectory() as temp_dir:
-        os.system(f'cp {cc_mask_file} {temp_dir}/tmp_mask.bmp')
-        # 2--bmp
-        geocode_back(f'{temp_dir}/tmp_mask.bmp', f'{temp_dir}/tmp_mask_geo.bmp', mwidth, lt, demw, demn, 2)
-        # 0--bmp
-        data2geotiff(f'{temp_dir}/tmp_mask_geo.bmp', f'{temp_dir}/tmp_mask_geo.tif', dempar, 0)
+        os.system(f'cp {cc_file} {temp_dir}/tmp_mask')
+        # 2--SUN raster/BMP/TIFF, 0--FLOAT (default)
+        geocode_back(f'{temp_dir}/tmp_mask', f'{temp_dir}/tmp_mask_geo', mwidth, lt, demw, demn, 0)
```

Comment:
Is the copy necessary? I think you can just do:
```suggestion
        # 2--SUN raster/BMP/TIFF, 0--FLOAT (default)
        geocode_back(cc_file, f'{temp_dir}/tmp_mask_geo', mwidth, lt, demw, demn, 0)
```